### ISSUE TITLE
Playback of Odyssee/LBRY Videos

### ISSUE THREAD

**Issue by ghost** [graham-walker/youtube-dl-react-viewer#25]
# Versions
I'm running version 1.30 with the provided docker compose file on Debian 12. Client browser is Firefox 115 on Windows.

# Description
Downloading and playback of youtube works. Downloading from odysse/lbry works, but when clicking on a video the page just goes dark.
The Firefox log provides the error message "this.props.videos is undefined" (see screenshot).

# How to replicate:
- Pull a fresh youtube-dl-react-viewer via Docker
- start a job for any odyssee video (e.g. https://odysee.com/@Odysee:8/getyouryoutubechannelonodysee:5)
- wait for it to download and verify by starting video from the download folder
- try to watch video on playback page

# Firefox log
![grafik](https://github.com/graham-walker/youtube-dl-react-viewer/assets/29985600/2c987ab8-f42c-4ce2-9e46-df4dd60dbca8)

**Comment by graham-walker** [comment]
I have pushed commit e25dbf2 that should resolve this issue. If you use the Docker installation method from that commit Odysee videos should play.

The issue occurred because Odysee does not return uploader metadata, only channel metadata, and the fallback properties were not configured correctly in youtube-dl-react-viewer to handle this.

While this commit allows Odysee videos to play, it does not fix the other related issue I found, which is  `channel_id` being used instead of `channel` as the name for videos that do not have uploader metadata. I will be working on fixing that next.
